clients/python/README.md

```diff
@@ -102,6 +102,30 @@ There are caveats to be noted when using this method:
     )
     ```
 
+## Advanced use-cases
+
+### Using Model Registry Python Client with newer Python versions
+
+> [!CAUTION]
+> The mechanism described in this section is a temporary workaround and likely will never be supported.
+> This workaround is ONLY applicable if your Python/Notebook project does NOT make use of MLMD directly or indirectly.
+
+<!-- a longer-term plan to address this ties to the investigations to rebase this client on top of MR REST api directly,
+so to avoid having to wrap the MLMD Wheel. See more: https://github.com/kubeflow/model-registry/pull/59 -->
+
+This project _currently_ wraps Google's MLMD and aligns with its supported Python versions (see more [here](https://pypi.org/project/ml-metadata/#files)),
+hence the reason of this project's [specified constraints](https://github.com/kubeflow/model-registry/blob/8d77c13100c6cc5a9465d4293403114a3576fdd7/clients/python/pyproject.toml#L14).
+
+To circumvent this current limitation, **and only IF your Python/Notebook project does NOT make use of MLMD directly or indirectly**,
```

Comment:
The wording is confusing for me is here, it assumes user knows more about how mlmd works, maybe expand on it?

Reply:
> it assumes user knows more about how mlmd works

this section is titled "Advanced use-cases", and I can't explain why a given Python version is required for the MR client without explaining implementation details here.

I'll see if I can find some alternative wordings...

Reply:
reworded with f579d08

Reply:
thank you